src/pheval/utils/semsim_utils.py

```diff
@@ -0,0 +1,86 @@
+"""
+Contains all pheval utility methods
+"""
+
+
+import click
+import numpy
+import pandas as pd
+import plotly.express as px
+
+import pheval.utils.file_utils as file_utils
+
+
+def filter_non_0_score(data: pd.DataFrame, col: str) -> pd.DataFrame:
+    """Gets rid of data with a score of 0 in the selected column
+
+    Args:
+        data (pd.DataFrame): Dirty dataframe
+        col (str): Column to be filtered
+
+    Returns:
+        pd.DataFrame: Filtered dataframe
+    """
+    return data[data[col] != 0]
+
+
+def parse_semsim(df: pd.DataFrame, cols: list) -> pd.DataFrame:
+    """Parses semantic similarity profiles converting the score column as a numeric value and dropping the null ones
+
+    Args:
+        df (pd.DataFrame): semantic similarity profile dataframe
+        cols (list): list of columns that will be selected on semsim data
+
+    Returns:
+        pd.Dataframe: parsed semantic similarity dataframe
+    """
+    df[cols[-1]] = pd.to_numeric(df[cols[-1]], errors="coerce")
+    df.replace("None", numpy.nan).dropna(subset=cols[-1], inplace=True)
+    return df
+
+
+def diff_semsim(data1: pd.DataFrame, data2: pd.DataFrame, score_column: str) -> pd.DataFrame:
```

Comment:
I think it would be more useful to give variables more meaningful names, i.e. instead of data1 and data2 use semsim1 and semsim2